app/main/views/search.py

```diff
@@ -226,10 +228,26 @@ def supplier_search():
         for tag in sorted(tags):
             services[tag] = True
         seller_type = details.get('seller_type', {})
-
         is_recruiter = details.get('is_recruiter', False)
-        if is_recruiter == 'true' and seller_type and 'recruiter' not in seller_type.keys():
-            seller_type['recruitment'] = True
+        recruiter = details.get('recruiter', {})
+
+        # consultant only badge to show in seller catalogue
+        if is_recruiter == 'false' and seller_type and 'consultant' not in seller_type.keys():
```

Comment:
Similar to the API comment, I have added ` is_recruiter`  instead of  `recruiter` so all consultant sellers will show their consultant badge